internal/core/application/blockchain_listener.go

```diff
@@ -391,9 +391,10 @@ func (b *blockchainListener) checkMarketAccountFundings(ctx context.Context, acc
 			}
 			log.Warnf("%s asset is missing for market %d", asset, accountIndex)
 		case 2:
+			baseAsset := config.GetString(config.BaseAssetKey)
```

Comment:
at shi point, wouldn't be better to be a filed of the `blockchainListener` itself? So we can entirely decouple from config